notary/content.md

```diff
@@ -0,0 +1,89 @@
+%%LOGO%%
+
+# Contents
+
+- [How to use this repository](#how_to_use_this_repository)
+- [Notary Server](#notary_server)
+- [Notary Signer](#notary_signer)
+- [Database Migrations](#database_migrations)
+
+# How to use this repository<a name="how_to_use_this_repository"></a>
```

Comment:
Unfortunately the Docker Hub does not currently render any HTML and doesn't even automake anchors for headings like github. :crying_cat_face: